pyproject.toml

```diff
@@ -20,21 +20,32 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
+
 exclude = ["geoh5py/handlers/*", "geoh5py/interfaces/*"]
 
+include = [
+    "COPYING",
+    "COPYING.LESSER",
+    "LICENSE",
+    "README.rst",
+    "THIRD_PARTY_SOFTWARE.rst",
+]
+
 [tool.poetry.dependencies]
 python = "^3.8,<3.11"
-Pillow = "10.1.0"
+
 h5py = "^3.2.1"
 numpy = "!=1.19.4, ~1.23.5"
+Pillow = "^10.1.0"
 
 
 [tool.poetry.group.dev.dependencies]
+Pygments = "*"
 lockfile = "^0.12.2"
 pylint = "*"
 pytest = "*"
 pytest-cov = "*"
-scipy = "^1.4.1"
+scipy = "~1.10.1"
```

Comment:
@domfournier , tests are failing. Could you please give it a look?
Maybe due to the change of scipy version here?

Note that SciPy is only used in tests for this repo. I have changed its version, so that tests run with that same version as the one we ship in the runtime environment

Reply:
thanks @domfournier for pointing at Pillow. I will constrain its version harder.

Reply:
That's actually a PIL issue. We need to pin it back for now.
To addressed on https://mirageoscience.atlassian.net/browse/GEOPY-1456